cfme/services/catalogs/catalog_item.py

```diff
@@ -152,7 +152,7 @@ def create(self):
                                'select_catalog': catalog.name,
                                'select_dialog': dialog.name,
                                'select_orch_template': self.orch_template,
-                               'select_provider': self.provider,
```

Comment:
Just checking,

Before self.provider would have been a provider object no? Or perhaps I am thinking of another place. I remember recently we changed a very similar call so that it no longer took the ```provider``` and ```provider_type``` args, but just the ```provider``` object. This is much more preferable as then you can use ```provider.name``` or ```provider.type``` when needed.